keps/sig-auth/20190607-certificates-api.md

```diff
@@ -234,7 +261,31 @@ Cluster admins can either:
 1. grant signer-specific approval permissions using roles they define
 2. grant signer-specific approval permissions using the bootstrap roles starting in 1.18
 3. disable the approval-authorizing admission plugin in 1.18 (if they don't care about partitioning approver rights)
- 
+
+### API changes between v1beta1 and v1
+
+`spec.signerName` validation:
+
+* required, not defaulted
+* `kubernetes.io/legacy-unknown` is disallowed when creating new CSR objects
+
+`status.certificate` validation:
```

Comment:
Should we validate that no certificates are self signed (i.e. rootless chain)? You mentioned that you want to avoid using this as a trust distribution mechanism.

Reply:
I wasn't planning to add tight signer chain validation at the API level, since multiple certificate blocks in this field can be used for purposes other than intermediates by other signerNames.

Reply:
I could see enforcing that in kube-controller-manager when determining what intermediate chains to accept/append